.github/actions/checkout-helm-versions/action.yml

```diff
@@ -0,0 +1,39 @@
+# This action supports helmVersions reporting in the TestRunner BigQuery schema. For example,
+# https://console.cloud.google.com/bigquery?cloudshell=false&project=terra-kernel-k8s&ws=!1m10!1m4!4m3!1sterra-kernel-k8s!2ssimple_stream_dataset!3ssimple_streamtable!1m4!4m3!1sterra-kernel-k8s!2ssimple_stream_dataset!3sSUMMARY_testRun
+name: 'checkout-helm-versions'
```

Comment:
It looks like this action is only used to get two files from this repo:
```
        "base-file-path": "terra-helmfile/versions/app/dev.yaml",
         "override-file-path": "terra-helmfile/environments/live/perf.yaml"
```

If so. it might be easier to use `curl` to extract these specific files using the raw GitHub file API. An example of how to do this for a private repo is here: https://stackoverflow.com/a/22314262